utilities/pipelines/publish/helper/Get-ModuleVersionChange.ps1

```diff
@@ -34,8 +34,20 @@ function Get-ModuleVersionChange {
         [string] $VersionFilePath
     )
 
+    git remote add 'upstream' 'https://github.com/Azure/bicep-registry-modules.git' 2>$null # Add remote source if not already added
+    git fetch 'upstream' 'main' -q # Fetch the latest changes from upstream main
```

Comment:
Are these needed if we are in upstream already?

Reply:
Let's have this merged and test one pipeline from upstream main to rule any doubt out